Get and delete documents by filter (#236)
meilisearch/specifications: text/0060-tasks-api.md

| name                | description                          |
|---------------------|--------------------------------------|
| providedIds         | Number of provided document ids.     |

> | originalFilter      | The filter used to delete documents. |

Reviewer: This field is only visible when deleting tasks with `POST` - `indexes/:index_uid/documents/delete`, right?

Author: Hey yes! I'll add it to the spec!

> | originalFilter      | The filter used to delete documents. Only present if the deletion provided a filter. |

Reviewer 2: ```suggestion
| originalFilter      | The filter used to delete documents. `null` if `filter` was not used for the deletion request. |
```
